web/pgadmin/tools/sqleditor/__init__.py

```diff
@@ -761,6 +762,22 @@ def start_view_data(trans_id):
 
     if status and conn is not None and \
             trans_obj is not None and session_obj is not None:
+
+        # Fetch the Namespace Name and object Name
+        sql_path = 'sqleditor/sql/#{0}#'.format(manager.version)
```

Comment:
Wouldn't the query execute give an error? Like table does not exist?

Reply:
We fixed this issue earlier but the fix was not right #6138.

Reply:
I mean, if the object does not exist in the database then query tool itself will give error in the messages tab?